xarray/backends/api.py

```diff
@@ -1310,6 +1310,7 @@ def to_zarr(
     dataset: Dataset,
     store: Union[MutableMapping, str, Path] = None,
     chunk_store=None,
+    storage_options: Dict[str, str] = None,
```

Comment:
Should this go at the end, or we make most of these kwarg-only? At the moment it would be a silent failure if someone is using kwargs